api/test/integration/conftest.py

```diff
@@ -170,14 +170,19 @@ def check_health(interval: int = 10, node_type: str = 'manager', agents: list =
                 f"docker inspect env_wazuh-{node}_1 -f '{{{{json .State.Health.Status}}}}'", shell=True)
             if not health.startswith(b'"healthy"'):
                 return False
-        return True
     elif node_type == 'agent':
         for agent in agents:
             health = subprocess.check_output(
                 f"docker inspect env_wazuh-agent{agent}_1 -f '{{{{json .State.Health.Status}}}}'", shell=True)
             if not health.startswith(b'"healthy"'):
                 return False
-        return True
+    elif node_type == 'nginx-lb':
+        health = subprocess.check_output(
+            f"docker inspect env_nginx-lb_1 -f '{{{{json .State.Health.Status}}}}'", shell=True)
+        if not health.startswith(b'"healthy"'):
+            return False
+
```

Comment:
I suggest adding an else: statement to return False or Raise an exception because the node_type is invalid.